.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
-# Docs 
-*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice 
+# Docs
+*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice
```

Comment:
GitHub doesn't like @cmwaters anymore - I think the account doesn't exist anymore, can you remove it?

Reply:
More like the account is no longer an owner.
That's quite the verification by Github!

Reply:
Done.